### ISSUE TITLE
Investigate use of nox for automated test runs on multiple recent versions

### ISSUE THREAD

**Issue by elray1** [reichlab/reichlab-python-template#9]
nox link: https://nox.thea.codes/en/stable/

e.g., maybe we could include a github CI workflow file that runs tests using the most recent python version as well as the lowest one matching the `requires-python` setting in the pyproject.toml file?

**Comment by bsweger** [comment]
Agree--this would be a good feature. I was planning (at some point) to investigate nox for cladetime (or maybe you'll do it first on your new package). 

Once one of us knows what we're doing, we should definitely add it to our Python template.

**Comment by bsweger** [comment]
Closing this--python-app-template is no longer up to date.
